jquery.h5-lightbox.js

```diff
@@ -32,25 +34,33 @@ $.fn.h5lightbox = function( options ) {
             };
 
         if ( isLightbox( imgLarge ) ) {
-            $this.wrap( "<a class='" + opt.wrapperClass + "' href='" + imgLarge + "'></a>" );
+                $this.wrap("<a class='" + opt.wrapperClass + "' href='" + imgLarge + "'" + "' title='" + altText + "' data-page='" + page + "'></a>");
         }
 
         // lightbox implementation
         $( "a." + opt.wrapperClass ).click(function( e ) {
             e.preventDefault();
-            var lightbox,
-            lbImageHref = $( this ).attr( "href" );
+                    var lbImageHref = $(this).attr("href"),
+                    lbImageAlt = $(this).attr("title"),
+                    lbPage = $(this).attr("data-page");
 
             if ( $( "#lightbox" ).length ) {
-                $( "#lightbox-img" ).html( "<img src='" + lbImageHref + "' />" );
+                    $("#lightbox-img").html("<img class='large' src='" + lbImageHref + "'' alt='" + lbImageAlt + "' />");
                 $( "#lightbox" ).show();
             } else {
-                lightbox =
+                    var lightbox =
                     "<div id='lightbox'>" +
-                        "<p>Click to close</p>" +
+                        "<p>Click anywhere to close</p>" +
                         "<div id='lightbox-img'>" +
-                            "<img src='" + lbImageHref + "' />" +
+                        "<img class='large' src='" + lbImageHref + "'' alt='" + lbImageAlt + "' />" +
                         "</div>" +
+                        "<a href='https://www.pinterest.com/pin/create/button/?url=" + encodeURIComponent(lbPage) + "&media=" + encodeURIComponent(location.protocol + "//" + location.hostname + "/" + lbImageHref) + "&description=" + encodeURIComponent(lbImageAlt) + "'" +
```

Comment:
I'd like to leave this pretty generic, so markup specific to a third party like Pinterest isn't going to make it into the plugin core, but it's interesting to see it in use here.